RELEASENOTES.md

```diff
@@ -13,6 +13,34 @@ Have new ideas? Need help with setting up your node? Come visit us [here](https:
+- **Support Debian 11 Bullseye**: This is the first release that is compatible with Debian Linux version 11.x, "Bullseye." The .deb packages now use absolute paths only, for compatibility with Bullseye's stricter package requirements. ([#3909](https://github.com/ripple/rippled/pull/3909))
+- **Improve Cache Performance**: A new container type was implemented that behaves similarly to std::unordered_map, but which has multiple distinct partitions that can be concurrently accessed by multiple threads. This container is now used as storage for several caches within rippled. These caches periodically remove old entries to limit their memory usage. This removal process referred to as “sweeping,” holds a lock while it executes. This lock prevents any other activities on the cache from being locked, such as required for access to the open ledger and performing necessary activities for consensus. The new container type allows multiple threads to concurrently “sweep” each partition of the underlying new container type. This greatly reduces the time in which the lock is held and therefore increases the stability and performance of the application.
+
+### Bug Fixes
+
+- **Clarify the safety of NetClock::time_point arithmetic**: * NetClock::rep is uint32_t and can be error-prone when   used with subtraction. * Fixes [#3656](https://github.com/ripple/rippled/pull/3656)
+- **Fix out-of-bounds reserve, and some minor optimizations**
+- **Fix nested locks in ValidatorSite**
+- **Fix clang warnings about copies vs references**
+- **Fix reporting mode build issue**
+- **Fix potential deadlock in Validator sites**
+- **Use libsecp256k1 instead of OpenSSL for key derivation**: The deterministic key derivation code was still using calls to OpenSSL. This replaces the OpenSSL-based routines with new libsecp256k1-based implementations
+- **Improve NodeStore to ShardStore imports**: This runs the import process in a background thread while preventing online_delete from removing ledgers pending import
+- **Simplify SHAMapItem construction**: The existing class offered several constructors which were mostly unnecessary. This eliminates all existing constructors and introduces a single new one, taking a `Slice`. The internal buffer is switched from `std::vector` to `Buffer` to save a minimum of 8 bytes (plus the buffer slack that is inherent in `std::vector`) per SHAMapItem instance.
+- **Redesign stoppable objects**: Stoppable is no longer an abstract base class, but a pattern, modeled after the well-understood `std::thread`. The immediate benefits are less code, less synchronization, less runtime work, and (subjectively) more readable code. The end goal is to adhere to RAII in our object design, and this is one necessary step on that path.
+- **Serialize metadata a blob in tx RPC when the binary flag is true**
```

Comment:
Wasn't this already fixed in a previous version?

Reply:
Which one are you referring to @mDuo13

Reply:
> - **Serialize metadata a blob in tx RPC when the binary flag is true**

This line should be removed because it was [part of 1.7.2](https://xrpl.org/blog/2021/rippled-1.7.2.html) that got reverse-merged into the 1.8.0 line. In 1.7.2 it was:

>** Properly encode metadata from tx RPC command:** Fixes a problem where transaction metadata in the tx API method response would be in JSON format even when binary was requested. (7311629)